mass/adapters/inbound/fastapi_/models.py

```diff
@@ -49,3 +49,4 @@ def no_duplicate_fields(cls, parameters: list[SortingParameter]):
         all_sort_fields = [param.field for param in parameters]
         if len(set(all_sort_fields)) < len(all_sort_fields):
             raise ValueError("Sorting parameters cannot contain duplicate fields")
+        return parameters
```

Comment:
Maybe add return type to the method signature

Reply:
Thanks, added that in [647c14d](https://github.com/ghga-de/mass/pull/16/commits/647c14ddef2e4fc9be0b3f8a627a35968d8b4976)